staticfiles/templates/contributors/home.html

```diff
@@ -1,235 +1,242 @@
 <div class="row text-center" ng-controller="HomeController as vm">
-    <div class="row">
```

Comment:
I have no idea. I didn't delete anything, only added my link. It looks like everything is added back in at the bottom